app/helpers/administrate/application_helper.rb

```diff
@@ -43,10 +44,10 @@ def model_from_resource(resource_name)
       dashboard.try(:model) || resource_name.to_sym
     end
 
-    def display_resource_name(resource_name)
+    def display_resource_name(resource_name, opts = {})
       dashboard_from_resource(resource_name).resource_name(
-        count: PLURAL_MANY_COUNT,
-        default: default_resource_name(resource_name),
+        count: opts[:singular] ? SINGULAR_COUNT : PLURAL_MANY_COUNT,
+        default: default_resource_name(resource_name, !!opts[:singular]),
```

Comment:
In which situation is this used? I'm trying to reproduce it, but I can't get it to work :-/

I'm starting to think perhaps we should have specs for this, but I'm not sure how to best do it...

Reply:
this change is to recreate the behaviour of index.html.erb:47 as it was before the change to allow for translation, as such it is already tested in feature tests of the links generated on the adim/log_entry page of the example app.

Reply:
we can definitely look into making unit tests for this too though

Reply:
Done 👍